src/main/resources/lib/hudson/test/table.jelly

```diff
@@ -83,7 +83,12 @@ THE SOFTWARE.
                     </j:forEach>
                 </td>
                 <td class="jenkins-mobile-hide">
-                    <a href="${rootURL}/${f.failedSinceRun.url}">${f.age}</a>
+                    <j:choose>
+                        <j:when test="${f.age == 0 or f.skipped}">0</j:when>
```

Comment:
could also not show it when its 0.

This could do with a UI rework imo, a table isn't the best way of laying this out.